### PR TITLE
Chain / reporting data categories: add cumulative area column

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: server/modules/category/service/categoryService.js [hunk 1]
```diff
@@ -112,6 +112,86 @@ export const fetchSamplingPointData = async ({ surveyId, levelIndex = 0, limit,
...
+      {}
+    ),
+    ...(Category.isReportingData(category)
+      ? {
+          area_cumulative: item.areaCumulative,
+        }
+      : {}),
+  }))
+}
+
```
Comment (ramirobg94) [inline server/modules/category/service/categoryService.js:194]: could you use extraction to simplify this code? it is hard to follow

**Review by ramirobg94 — CHANGES_REQUESTED** [review]

File: server/modules/category/service/categoryService.js (hunk 1 above)
Comment (SteRiccio) [reply server/modules/category/service/categoryService.js:194]: of course, everything is possible

Comment (SteRiccio) [reply server/modules/category/service/categoryService.js]: moved to a separate file, a bit better but still "complex"...
